go_lib/cloud-data/go.mod

```diff
@@ -3,45 +3,52 @@ module github.com/deckhouse/deckhouse/go_lib/cloud-data
 go 1.19
 
 require (
-	github.com/alecthomas/kingpin v2.2.6+incompatible
-	github.com/prometheus/client_golang v1.14.0
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/golang/protobuf v1.5.2 // indirect
+	github.com/imdario/mergo v0.3.15 // indirect
 	github.com/sirupsen/logrus v1.9.0
-	k8s.io/apimachinery v0.26.3
-	k8s.io/client-go v0.26.3
+	gopkg.in/yaml.v2 v2.4.0 // indirect
+	k8s.io/apimachinery v0.25.5
```

Comment:
we are using 0.25 in Deckhouse `go.mod` and we should use same in deeps too.
I'm afraid to update deps in Deckhouse `go.mod` because we update k8s deps with updating k8s version